src/logic/basic.lean

```diff
@@ -5,6 +5,7 @@ Authors: Jeremy Avigad, Leonardo de Moura
 -/
 import tactic.doc_commands
 import tactic.reserved_notation
+import tactic.cache
```

Comment:
Is this intentional?

Reply:
Ah yes I did that because the `introI` tactic wasn't working in `logic.basic`. This was all from putting the `subsingleton_iff` lemma in there which maybe is not what should be done, I was just trying to see if this fixed other problems,